index.js

```diff
@@ -10,9 +10,10 @@ utils.setConfig({
 function interpolate(object, keyPath) {
     var keys = keyPath.split('.');
 
-    return _(keys).reduce(function(res, key) {
+    return !_.isEmpty(
```

Comment:
This will change the function definition, please revert this change. You can instead update `getFloatingNavItems` by replacing

```js
     return _(floating_nav_items).filter(function(item) {
         return interpolate(resume, item.requires);
     });
```

with

```js
     return _(floating_nav_items).filter(function(item) {
         var value = interpolate(resume, item.requires);

         return _.isEmpty(value);
     });
```

Which essentially is your change but at a different place